subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dependencies/AbstractExternalModuleDependency.java

```diff
@@ -56,7 +55,6 @@ public AbstractExternalModuleDependency(ModuleIdentifier module, MutableVersionC
 
     protected void copyTo(AbstractExternalModuleDependency target) {
         super.copyTo(target);
-        DeprecationLogger.whileDisabled(() -> target.setForce(isForce()));
```

Comment:
Do we want to copy the force state still, for enforced platforms?

Reply:
This isn't relevant with the state being removed completely now.